### ISSUE TITLE
Embedded Python - local directories in pythonXX._pth

### ISSUE THREAD

**Issue by 7c080445-ae8a-4f06-b650-03b37a771d0d** [python/cpython#88839]
BPO | [44673](https://bugs.python.org/issue44673)
--- | :---
Nosy | @zooba

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2021-07-19.13:42:09.290>
labels = ['type-bug', '3.9']
title = 'Embedded Python - local directories in pythonXX._pth'
updated_at = <Date 2022-01-20.00:23:34.012>
user = 'https://bugs.python.org/emve'
```

bugs.python.org fields:
```python
activity = <Date 2022-01-20.00:23:34.012>
actor = 'steve.dower'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = []
creation = <Date 2021-07-19.13:42:09.290>
creator = 'emve'
dependencies = []
files = []
hgrepos = []
issue_num = 44673
keywords = []
message_count = 2.0
messages = ['397797', '410994']
nosy_count = 2.0
nosy_names = ['steve.dower', 'emve']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue44673'
versions = ['Python 3.9']
```

</p></details>

**Comment by lwcorp** [comment]
> You may want to try adding a startup file (search for PYTHONSTARTUP) that modifies sys.path directly.
Are you sure it's supported in Embedded Python? Most environmental variables aren't, and it doesn't seem to work for me.

**Comment by zooba** [comment]
You're right, that was a bad suggestion. It's really just outside the spec for what the distro is meant to do, so you'd want a regular install (either from python.org or from the Store) and a virtual environment if it's meant for multiple projects in non-fixed locations.

**Comment by lwcorp** [comment]
> You're right, that was a bad suggestion. It's really just outside the spec for what the distro is meant to do, so you'd want a regular install (either from python.org or from the Store) and a virtual environment if it's meant for multiple projects in non-fixed locations.

Some suggest a third alternative under the claim that the "real" portable version is [nuget](https://www.nuget.org/packages/python/) - see [whole discussion](https://discuss.python.org/t/add-a-command-line-parameter-to-add-folders-to-sys-path-needed-for-embedded-python/28426/10).

**Comment by zooba** [comment]
That "some" is me, so yes, I agree 😉 

CPython doesn't properly handle being used portably, as discussed in that thread. The embeddable distro is safe, the Nuget package is only safe when there are no normal installations (but still normally works okay), and normal installations are safe.
